### PR TITLE
[Level 2] - Responsive styles

### PR DESCRIPTION
**Pull request by jeffgaynor**
- Setup responsive top nav
- Adjusted layout and page styles for responsive
- Added HTML seo meta data
- Added font styles to theme

image

image

### CODE REVIEW

File: components/Layout/Page.tsx [hunk 1]
```diff
@@ -42,6 +47,7 @@ export const Page = ({ children }: PageProps) => {
         <meta name="description" content="These are yummy recipes." />
         <meta name="keywords" content="food,appetizers,desserts,drinks" />
         <link rel="icon" href="/favicon.ico" />
+        <base href="/" />
```
Comment (iAdramelk) [inline components/Layout/Page.tsx]: Q: Why use base here?

File: components/Layout/Nav/Nav.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import Link from 'next/link';
+import { List, ListItem, NavButton, NavIcon, NavTrigger } from './Nav.styles';
+
+export const Nav = () => {
+  return (
+    <nav role="navigation">
+      <NavButton className="nav-btn" type="checkbox" id="nav-btn" />
```
Comment (iAdramelk) [inline components/Layout/Nav/Nav.tsx]: Q: Why use `className`s here?

**Review by iAdramelk — COMMENTED** [review]
Added a couple of questions!

File: components/Layout/Page.tsx (hunk 1 above)
Comment (jeffgaynor) [reply components/Layout/Page.tsx]: That was something I added when first setting up pages, and is probably not needed. I'll remove.

File: components/Layout/Nav/Nav.tsx (hunk 2 above)
Comment (jeffgaynor) [reply components/Layout/Nav/Nav.tsx]: Ah, looks like I missed removing this when cleaning up. Good catch!

**Comment by jeffgaynor** [comment]
> Added a couple of questions!

@iAdramelk Thanks for those! I just removed the remaining classNames in the top nav.

**Review by C-STYR — APPROVED** [review]

**Pull request merged**